tests/functional/clients/standalone/package_list/test_debian.py

```diff
@@ -7,29 +7,41 @@
 
 class TestDebianPaths:
 
-    @pytest.mark.skipif(sys.platform == "darwin", reason="no dpkg on OSX")
     @pytest.mark.parametrize('path', path_params(debian.pkgfiles_all))
     def test_pkgfiles_all(self, analyzed_data, path):
         result = analyzed_data("stretch-slim")
         pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgfiles.all']['base']
         assert pkgs.get(path) == 'DPKGFILE'
 
-    @pytest.mark.skipif(sys.platform == "darwin", reason="no dpkg on OSX")
     @pytest.mark.parametrize('pkg,version', [pytest.param(pkg, version, id=pkg) for pkg, version in debian.pkgs_all.items()])
     def test_pkgs_all(self, analyzed_data, pkg, version):
         result = analyzed_data("stretch-slim")
         pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgs.all']['base']
         assert pkgs.get(pkg) == version
 
-    @pytest.mark.skipif(sys.platform == "darwin", reason="no dpkg on OSX")
     @pytest.mark.parametrize('pkg,metadata', debian.pkgs_allinfo.items())
     def test_pkgs_allinfo(self, analyzed_data, pkg, metadata):
         result = analyzed_data("stretch-slim")
         pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgs.allinfo']['base']
-        loaded = json.loads(pkgs.get(pkg, '{}'))
+        loaded = pkgs.get(pkg, {})
+
+        # a separate test exists just for the licenses attribute, leave this out of the assertion
+        loaded = dict(loaded)
+        loaded.pop('license')
+        metadata = dict(metadata)
+        metadata.pop('license')
+
         assert loaded == metadata
 
-    @pytest.mark.skipif(sys.platform == "darwin", reason="no dpkg on OSX")
+    @pytest.mark.parametrize('pkg,metadata', debian.pkgs_allinfo.items())
+    def test_pkgs_allinfo_license(self, analyzed_data, pkg, metadata):
+        result = analyzed_data("stretch-slim")
+        pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgs.allinfo']['base']
+        loaded = pkgs.get(pkg, {})
+        actual = sorted(loaded['license'])
+        expected = sorted(metadata['license'])
```

Comment:
this sorting is causing issues/failures:

```
        actual = sorted(loaded['license'])
        expected = sorted(metadata['license'])
>       assert actual == expected
E       AssertionError: assert ['U', 'k', 'n...'n', 'o', ...] == [' ', '-', '-...'1', '2', ...]
E         At index 0 diff: 'U' != ' '
E         Right contains 7 more items, first extra item: 'G'
E         Full diff:
E         - [' ', '-', '-', '.', '1', '2', '2', 'G', 'G', 'L', 'L', 'L', 'P', 'P']
E         + ['U', 'k', 'n', 'n', 'n', 'o', 'w']
```

Reply:
added the split and migrated to set 👍